docs/modules/ROOT/pages/how-tos/testing/integration/periodic-integration-tests.adoc

```diff
@@ -0,0 +1,114 @@
+= Triggering Periodic Integration Tests
+
+In {ProductName}, you can run integration tests on a schedule using Kubernetes CronJobs.
+
+== Procedure
+
+To trigger a periodic Integration Test, complete the following steps:
+
+. Create an integration test by following the xref:how-tos/testing/integration/creating.adoc[Creating a integration test guide].
+
+. Create RBAC rules in your tenant to access Konflux snapshots in your tenant.
+
+. Create a link:https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[CronJob] that will trigger periodic Integration Tests via xref:how-tos/testing/integration/rerunning.adoc[snapshot labels].
+
+=== Procedure with example
+
+This procedure assumes that you already have an integration test created and a ServiceAccount available in your namespace.
+
+. Create a `Role` and `RoleBinding` to grant the necessary snapshot access to the ServiceAccount, which will be used to trigger the periodic Integration Tests
+
+[source,yaml]
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: periodic-jobs-role
+  namespace: <tenant-name>
+rules:
+  - verbs:
+      - get
+      - watch
+      - list
+      - update
+      - patch
+    apiGroups:
+      - appstudio.redhat.com
+    resources:
+      - snapshots
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: periodic-jobs-rolebinding
+  namespace: <tenant-name>
+subjects:
+  - kind: ServiceAccount
+    name: <service-account-name>
+    namespace: rhtap-build-tenant
+roleRef:
+  kind: Role
+  name: periodic-jobs-role
+  apiGroup: rbac.authorization.k8s.io
+
+. Create a CronJob that will trigger a Periodic via snaphot labels
+
+[source,yaml]
```

Comment:
Can you describe what this CronJob does? You can use the `<.>` keys to refer to specific points in the yaml file if you want to describe it in footnotes (see other pages for reference).

Reply:
Oh! I will! Thank you

Reply:
@arewm Im about to move this cronjob example to tekton-integration-catalog. OR should I keep it here? Is to much content

Reply:
see https://github.com/flacatus/tekton-integration-catalog/blob/samples/samples/periodic-integration-tests/periodic-integration-tests.yaml